client/blocks/google-my-business-stats-nudge/actions.js

```diff
@@ -0,0 +1,17 @@
+/** @format */
+
+/**
+ * Internal Dependencies
+ */
+import { getPreference } from 'state/preferences/selectors';
+import { savePreference } from 'state/preferences/actions';
+
+export const dismissNudge = () => ( dispatch, getState ) => {
+	const nudgePreference = getPreference( getState(), 'google-my-business-dismissible-nudge' );
+	return dispatch(
+		savePreference( 'google-my-business-dismissible-nudge', {
```

Comment:
Why `dispatch` and not just `return`?

Reply:
because of its signature: `() => ( dispatch, getState ) => {}` instead of `() => {}` which would require an action to be returned for redux-thunk to dispatch it.
This is because we need to access the state with `getState()`

Reply:
yep, you right, something got mixed in my mind 🤦‍♂️